docs/core/additional-tools/WCF-Web-Service-Reference-guide.md

```diff
@@ -0,0 +1,51 @@
+## Microsoft WCF Web Service Reference Provider
+
+Over the years, many Visual Studio developers have enjoyed the productivity that Add Service Reference provides when their .NET Framework projects needed to access Windows Communication Foundation (WCF) services. The WCF team is excited to introduce the WCF Connected Service for .NET Core and ASP.NET Core, a VS extension providing an experience like the Add Service Reference functionality. This tool retrieves metadata from a WCF service in the current solution, locally or on a network, and generates a .NET Core compatible source code file for a WCF client proxy that you can use to access the service.
```

Comment:
> ... that Add Service Reference provides when their .NET Framework projects needed to ...


It seems to me that either `provides` should be `provided` or `needed` should be `need`.

Reply:
> This tool retrieves metadata from a WCF service in the current solution, locally or on a network...

This sentence doesn't sound right.

Reply:
Yeah, maybe something like "This tool retrieves metadata from a WCF service in the current solution, a WSDL file at the local disk or a web service on a network...".

Reply:
fixed the 'locally' part, the rest is OK.